src/mlpack/tests/main_tests/preprocess_scale_test.cpp

```diff
@@ -0,0 +1,117 @@
+/**
+ * @file preprocess_scale_test.cpp
+ * @author Jeffin Sam
+ *
+ * Test mlpackMain() of preprocess_scale_main.cpp.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#define BINDING_TYPE BINDING_TYPE_TEST
+
+#include <mlpack/core.hpp>
+static const std::string testName = "PreprocessScale";
+
+#include <mlpack/core/util/mlpack_main.hpp>
+#include <mlpack/methods/preprocess/preprocess_scale_main.cpp>
+
+#include "test_helper.hpp"
+#include <boost/test/unit_test.hpp>
+#include "../test_tools.hpp"
+
+#include <cmath>
```

Comment:
I think this is already included in `mlpack/core.hpp`---correct me if I'm wrong.

Reply:
Yes, you are right.